src/icons/Warning.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import Colors from 'src/styles/colors'
+
+const Warning = ({ color = Colors.warningDark, size = 16 }: { color?: Colors; size?: number }) => (
```

Comment:
nit: using `function` is easier to read IMHO
```suggestion
export default function Warning({ color = Colors.warningDark, size = 16 }: { color?: Colors; size?: number }) (
```